src/libsyntax_ext/deriving/mod.rs

```diff
@@ -81,7 +81,10 @@ fn expand_derive(cx: &mut ExtCtxt,
                  mitem: &MetaItem,
                  annotatable: Annotatable)
                  -> Annotatable {
-    annotatable.map_item_or(|item| {
+    debug!("expand_derive: span = {:?}", span);
+    debug!("expand_derive: mitem = {:?}", mitem);
+    debug!("expand_derive: annotatable input  = {:?}", annotatable);
```

Comment:
Any reason for 3 debug printouts instead of one like 

```rust
debug!("expand_derive: span = {:?}, mitem = {:?}, annotatable = {:?}", span, mitem, annotatable);
```